src/libPMacc/include/cuSTL/container/CartBuffer.hpp

```diff
@@ -56,12 +62,13 @@ namespace container
  */
 template<typename Type, int T_dim, typename Allocator = allocator::EmptyAllocator,
                                   typename Copier = mpl::void_,
-                                  typename Assigner = mpl::void_>
-class CartBuffer
+                                  typename Assigner = bmpl::vector<bmpl::_1, bmpl::_2> >
```

Comment:
please add a comment to the doxygen docu that this `Assigner ` must support `apply2` and which elements were applied